src/usb_moded-dbus.c

```diff
@@ -41,6 +42,25 @@
 static DBusConnection *dbus_connection_sys = NULL;
 extern gboolean rescue_mode;
 
+/**
+ * Issues "sig_usb_config_ind" signal.
+*/
+static void usb_moded_send_config_signal(const char *section, const char *key, const char *value)
+{
+  if (dbus_connection_sys)
+  {
+	DBusMessage* msg = dbus_message_new_signal(USB_MODE_OBJECT, USB_MODE_INTERFACE, USB_MODE_CONFIG_SIGNAL_NAME);
+	if (msg) {
+		dbus_message_append_args(msg, DBUS_TYPE_STRING, &section,
+		                              DBUS_TYPE_STRING, &key,
+		                              DBUS_TYPE_STRING, &value,
```

Comment:
Does it make sense to make the value a D-Bus `DBUS_TYPE_VARIANT`? It would make marshalling the value harder here, and depending on how the consumer uses it. But then, it could avoid issues where converting to/from string is lossy, or where a certain key can have different value types.